stdlib/@tests/stubtest_allowlists/linux-py313.txt

```diff
@@ -1,5 +1,4 @@
 # TODO: triage these (new in py313)
-_stat.SF_SETTABLE
```

Comment:
Interesting, do you know why this allowlist entry gets to be removed with this change? What was the complaint before, do you know?

Reply:
Hmm, it wasn't very specific: https://github.com/python/typeshed/actions/runs/9921662631/job/27409865933?pr=12332

I'm honestly not sure, maybe this an issue in stubtest where `Literal[x]` and `CONST: Final = x` are treated differently?

Reply:
> Hmm, it wasn't very specific: [python/typeshed/actions/runs/9921662631/job/27409865933?pr=12332](https://github.com/python/typeshed/actions/runs/9921662631/job/27409865933?pr=12332)
> 
> I'm honestly not sure, maybe this an issue in stubtest where `Literal[x]` and `CONST: Final = x` are treated differently?

Yeah, I think stubtest does stricter verification of `Literal`-typed variables than it does with `Final`-typed variables that have default values. (It's something I've had on my to-do list for a while to improve... see https://github.com/python/mypy/pull/14568, which needs some work, it's not really going about things the right way right now). It comes down to the fact that `Final`- and `Literal`-typed variables have different internal representations in mypy.

What does stubtest say if you try removing the allowlist entries on the `main` branch? What are its complaints there?

Reply:
Let me give it a try.

Reply:
I get `_stat.SF_SETTABLE variable differs from runtime type Literal[-65536]`: https://github.com/python/typeshed/actions/runs/9943404744/job/27467158147?pr=12347

Reply:
Yes, it looks like it's three different values on the three different platforms we test in CI. That implies to me it should be `Final[int]` in the stubs rather than us writing the precise value in there (and we should add a comment saying that it varies across platforms)

Reply:
Will do!

Reply:
https://github.com/python/typeshed/pull/12332/commits/fc18549ac472080d19b720da619790a44a2bbab5